Documentation/QuickInstall/TheInstallTool/Index.rst

```diff
@@ -79,7 +79,36 @@ The Install Tool
       :class: with-shadow
 
       Install Tool in 1-2-3 mode, fifth step.
-
+      
```

Comment:
I agree, that we should mention introduction package in Install & Upgrade guide, the question is just where and how.

* Currently there already is a section: In the "In depth Installation" chapter: https://docs.typo3.org/m/typo3/guide-installation/10.4/en-us/In-depth/Distributions/Index.html#introduction-package We should be careful not to create too much duplicate information
* should the Introduction package really be in the "Install Tool" page? (when installing it without Composer, it makes sense. When installing it with Composer it probably does not). In that case we should probably add it to this page: https://docs.typo3.org/m/typo3/guide-installation/10.4/en-us/QuickInstall/Composer/Index.html. But it already exists in the In depth section: https://docs.typo3.org/m/typo3/guide-installation/master/en-us/In-depth/Distributions/Index.html#installing-distributions-with-composer